Libplanet.Net/Swarm.cs

```diff
@@ -858,7 +858,7 @@ CancellationToken cancellationToken
                     List<byte[]> payloads = blockMessage.Payloads;
                     _logger.Information(
                         "Received {Count} blocks from {Peer}",
-                        payloads.Count,
+                        payloads.Count / 2,
```

Comment:
Is this intended?

Reply:
Since payloads contains both of `Block` and `BlockCommit`, for now it represents two times of actual `Block` count. Saparating payload can be considered also, but for now applied this to fix error on logging.

Reply:
Actually, it's not related with this PR. just minor log fix.